templates/e2e-llm-workflows/src/utils.py

```diff
@@ -63,3 +64,20 @@ def get_dataset_file_path(dataset: Dataset):
         assert len(os.listdir(temp_path)) == 1, "The dataset should be written to a single file"
         dataset_file_path = f"{temp_path}/{os.listdir(temp_path)[0]}"
         yield dataset_file_path
+
+def set_key_value_in_config_file(config_path: str, key: str, value: str):
+    """Take a path to a .json config file and update the key to a value, then save the file"""
+    with open(config_path, 'r') as stream:
+        try:
+            loaded = yaml.safe_load(stream)
+        except yaml.YAMLError as exc:
+            print(exc)
+            exit()
```

Comment:
Since we're not handling the exception, how about just leaving it as is for the user to handle? 
```suggestion
    with open(config_path, 'r') as stream:
        loaded = yaml.safe_load(stream)
```